### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by redwing555**
This is an astrophysics conference website for the HTML  / CSS and JavaScript Capstone project.

➡️ Contains 2 pages: Homepage and About page 

➡️ The website is mainly mobile-first and responsive for desktop screens.

➡️It is done using the Microverse project specifications and HTML / CSS / Javascript best practices learner during the first module of the program.

➡️  This is the link for the video on Loom presenting my project:  [Loom recording of project presentation](https://www.loom.com/share/0f00df9801b6476f88af35e2e3d5a755 )

📔📔📔📔📔📔   Unfortunately the video on loom is only 5 mins and I didn't have enough time to go into HTML and CSS specifics about the website.  🙏 🙏 🙏 🙏 🙏

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+                    </p>
+
+                    <p class="conf-time">
+
+                        <strong>2021.09.15 (FRI) - 2021.09.20 (TUE)</strong> <br>
+                        <small class="location">@Hassan 2 University , Casablanca - Morocco.</small>
+
+
+                    </p>
+
```
Comment (DeliceLydia) [inline index.html]: - [ ] Please follow the given design on [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as per requirements you are allowed to personalize contents but should follow the design that means colors, layouts, background..
your page
bc

original
lay

File: index.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+            </section>
+
+            <section class="speakers">
+                <div>
+                    <h2>
+                        Featured speakers
+                    </h2>
+                    <div></div>
+                </div>
```
Comment (DeliceLydia) [inline index.html:164]: - [ ] Add some margin-bottom between the heading and the speaker this is how it looks on the desktop now
main

File: index.html [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+                    
+                    <div class="speaker-info">
+                        <h3>Stephen Hawking</h3>
+                        <p>director of research at the University of Cambridge</p>
+                        <div></div>
+                        <p>English theoretical physicist whose theory of exploding black holes drew upon both relativity theory and quantum mechanics.</p>
+                    </div>
+                    
+
+                </article> -->
```
Comment (DeliceLydia) [inline index.html]: - [ ] Please remove commented codes to make your codes clean

File: index.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+                        <p>Accomodation in nearest hotel for all our guests</p>
+                    </article>
+
+                    <a class="register" href="#"> Register for the conference</a>
+
+
+                </div>
+
+                <a href="#"><span class="see-program">SEE THE WHOLE PROGRAM</span> </a>
+
```
Comment (DeliceLydia) [inline index.html:155]: - [ ] On the desktop this link is not centered please work on that
cent

File: about.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                        <li><a class="mnitem" href="index.html#mainprogram">Accomodation</a></li>
+                        <li><a class="mnitem" href="index.html#fund">Funding</a></li>
+                        <li><a class="mnitem register-bar" href="#">Register</a></li>
+                        
+
+                    </ul>
+                </div>
+            </div>
+
+    </nav>
```
Comment (DeliceLydia) [inline about.html]: - [ ] On the desktop the navbar links are not all fitting on the page check out this image and make necessary changes
spa

File: README.md [hunk 6]
```diff
@@ -65,12 +70,7 @@ Feel free to check the [issues page](../../issues/).
...
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./MIT.md) licensed  and special credit to Cindy Shin (https://www.behance.net/adagio07) for this template under the Creative Commons license (https://creativecommons.org/licenses/by-nc/4.0/) .
```
Comment (DeliceLydia) [inline README.md]: - [ ] You can do something like special credit to [Cindy Shin](https://www.behance.net/adagio07) for this template under the [Creative Commons license](https://creativecommons.org/licenses/by-nc/4.0/) to look more cleaner.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hi @redwing555, 

Amazing job for real.. I can see the hard work from your project my major highlights for your project are:
- I liked how you explained each and everything you implemented in the video
- And you highlighted challenges you faced and I can see that you overcame them
- Your app is responsive
- PR is professional
- Commits messages are descriptive
Good job!

There are few issues you still need to address but don't mind you are almost there don't give up

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  I recommend that in your README when you are adding a link it doesn't look good when the link shows itself you can do something like []() you add link name and then the link itself will appear as a link I hope you get what I mean


### Cheers and Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @redwing555  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**